customer-onboarding-showcase/readme.md

```diff
@@ -0,0 +1,31 @@
+# Customer Onboarding Showcase
+This module holds the code for the Customer Onboarding Showcase. It is a showcase of the capabilities of the Miranum Framework 
+(IDE as well as Connect) framework. In the figure below you can see the customer onboarding process as well as the integrations
+which are written using the Miranum Connect Framework. 
+
+![Customer-Onboarding-Process](./images/process.png)
+
+Currently, this showcase only supports Camunda Platform 7 as a process engine. This can be easily changed by migrating the process
```

Comment:
I watched your presentation and I saw that you had c8 applications. So these exist, but not a c8 bpmn?

Reply:
I removed all of the C8 applications as of now, since they were not relevant to the demo and for the next upcoming ones. Imho it is just important to point it out. Also Miranum-IDE works with C7 only. 
It can be irritating for a viewer to see two modelling components in a demo.